### PR TITLE
CONCF-560 | Layout broken for long titles on FF

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/CONCF-560

https://bugzilla.mozilla.org/show_bug.cgi?id=1136818

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
@vforge @bognix

File: skins/oasis/css/core/PageHeader.scss [hunk 1]
```diff
@@ -237,6 +237,7 @@
 			@include align-self(flex-start);
 			@include flex(0 1 100%);
 			order: 0;
+			min-width: 0;
```
Comment (Warkot) [inline skins/oasis/css/core/PageHeader.scss]: CSS properties should be in alphabetical order. This rule is common and is also added to our [Guidelines repo](https://github.com/Wikia/guidelines). You can clone it and then install pre-commit hooks from there and/or enable scss-lint plugin in Php/WebStorm and point to config file from the repo.

Second thing is that when you're doing such ~~hacks~~ workarounds, add a comment why it's done. For someone who didn't work on the issue, who has no context, this line looks like something that does nothing and can be deleted. Please add the ticket number in the comment and a link to the Mozilla bug board, few words that it handles text wrapping would be nice, too.

Comment (vforge) [inline skins/oasis/css/core/PageHeader.scss]: I second @Warkot comment

**Comment by Warkot** [comment]
LGTM :+1:

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**